css/properties/font-smooth.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "font-smooth": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-smooth",
+          "support": {
+            "chrome": {
+              "version_added": "5"
```

Comment:
All flavors of Chrome implement this as `-webkit-font-smoothing`. I don't have numbers for Android and webview so use `true`.

Reply:
Fixed in 4151477ccfd37dfb96bfd8ace79a477a8f4d5f9d.